src/g3log.cpp

```diff
@@ -227,12 +227,13 @@ namespace g3 {
        */
       void pushFatalMessageToLogger(FatalMessagePtr message) {
          if (!isLoggingInitialized()) {
+            const auto& messagePtr = message.get();
```

Comment:
Please revert this. Using raw pointers is poor C++ practice and even if it's safe within this limited scope it can easily cause bad code creep. 

There's zero overhead (or completely negligible) with calling a function through unique_ptr compared to calling a function through raw pointer.

Reply:
Done.

Reply:
Actually. I misread your code. You are calling from the unique_ptr, just "unboxed" access to it from [MoveOnCopy](https://github.com/KjellKod/g3log/blob/d3f2d11ae603d1c68c3c29d164c3c7b8dbe56f27/src/g3log/moveoncopy.hpp). 

Regardless, you should revert this as 
1. this is only called in the case where the programmer has done a mistake, 
2. or ... the programmer is choosing to not follow setup installation best practices for an unknown reason. 

Regardless, it should be an exceptional case and should not be considered for any possible (?) performance improvement.

Reply:
Reverted anyway.